### PR TITLE
Write an entry for move()

### PR DESCRIPTION
**Pull request by E-Aghbari**
### Description

Codecademy assigned this issue for me to experience on how is it to be a contributor; thus, this is my first contribution into a repository on Github. Therefore, I have created a term entry for the `.move()` function in the `move` folder within the `table` folder. A metadata about the function has been written at the very top of the document with brief description of it I also explained the parameters of the function and what type of data type that is passed for each argument. Using an example, I showcased how the function work using two different tables.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/tables/terms/move/move.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+table.move(Table1, 1, #Table1, #Table2 + 1, Table2)
+
+print(table.concat(Table2))
+```
+
+
+In this example, the elements in `Table1` have been copied to `Table2` using the `.move()` function. Starting from index 1 in `Table1` until the end and then copying them in sequence to `Table2` starting from index 1.
+
+
+The above code will result in:
```
Comment (KTom101) [inline content/lua/concepts/tables/terms/move/move.md:48]:
```suggestion
The above code will result in:

```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@E-Aghbari I've added some changes to this entry.

File: content/lua/concepts/tables/terms/move/move.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.move()`** function copies the specified indices' values from the source table into the provided starting index of the destination table.
+
+## Syntax
+```lua
+table.move(a1, f, e, t, a2)
+```
```
[the remark this answers is not in the document]
Comment (E-Aghbari) [reply content/lua/concepts/tables/terms/move/move.md:20]: Wouldn't that result make the whole entry a block of code since ``` at bottom removed?

**Review by KTom101 — APPROVED** [review]
@E-Aghbari Please wait for a second review.

File: content/lua/concepts/tables/terms/move/move.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+`.move()` function expects five arguments where `a1` is the source table and `a2` is the destination table.
+
+`f` means _from_ (First index within `a1`).
+
+`e` means _end_ (Last index within `a1`).
+
+`t` means _to_ (First index within `a2`).
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:29]:
```suggestion
- `.move()` function expects five arguments where `a1` is the source table and `a2` is the destination table.
- `f` means _from_ (First index within `a1`).
- `e` means _end_ (Last index within `a1`).
- `t` means _to_ (First index within `a2`).
```

File: content/lua/concepts/tables/terms/move/move.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Example
+
+Consider the following tables:
+
+```lua
+local Table1 = {"🍎", "🍌", "🍇", "🍓", "🍉" }
+
+local Table2 = {}
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:38]:
```suggestion
local Table1 = {"🍎", "🍌", "🍇", "🍓", "🍉" }

local Table2 = {"🍎", "🍌"}
```

File: content/lua/concepts/tables/terms/move/move.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+table.move(Table1, 1, #Table1, #Table2 + 1, Table2)
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:40]:
```suggestion
table.move(Table1, 1, #Table1, 3, Table2)
```

File: content/lua/concepts/tables/terms/move/move.md (hunk 4 above)
Comment (caupolicandiaz) [reply content/lua/concepts/tables/terms/move/move.md:38]: I think the operation might be more obvious if the move is happening to a table that has some elements

File: content/lua/concepts/tables/terms/move/move.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+print(table.concat(Table2))
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:42]:
```suggestion
print(table.unpack(Table2))
```

File: content/lua/concepts/tables/terms/move/move.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+In this example, the elements in `Table1` have been copied to `Table2` using the `.move()` function. Starting from index 1 in `Table1` until the end and then copying them in sequence to `Table2` starting from index 1.
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:45]:
```suggestion
In this example, the elements in `Table1` have been copied to `Table2` using the `.move()` function. Starting from index 1 in `Table1` until the end and then copying them in sequence to `Table2` starting from index 3.
```

File: content/lua/concepts/tables/terms/move/move.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+The above code will result in:
+
+```lua
+🍎🍌🍇🍓🍉
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:50]:
```suggestion
🍎 🍌 🍎 🍌 🍇 🍓 🍉
```

File: content/lua/concepts/tables/terms/move/move.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+> **Note:** In Lua, indices start from 1; thus, the fourth argument is `#Table2 + 1` where `#Table2` is the length of the table. Therefore, `0 + 1` is the starting index of `Table2`.
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/move/move.md:53]:
```suggestion
> **Note:** In Lua, indices start from 1; thus, the next available index in `Table2` is index 3.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@E-Aghbari I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by E-Aghbari** [comment]
I have finished updating the entry, please have a look. Thank you. @caupolicandiaz

**Review by caupolicandiaz — APPROVED** [review]
@E-Aghbari @KTom101 LGTM

**Comment by E-Aghbari** [comment]
@caupolicandiaz Why does the CLA shows that i have not signed yet when I already have?
Thanks.

**Comment by E-Aghbari** [comment]
@caupolicandiaz It seems that I commited with different username Ebrahim Al Aghbari for some reason and the CLA refuses to carry on unless its signed but I dont know how to remove that commit. Is it possible to remove that commit and them commit again with same user?

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz It seems that I commited with different username Ebrahim Al Aghbari for some reason and the CLA refuses to carry on unless its signed but I dont know how to remove that commit. Is it possible to remove that commit and them commit again with same user?

I'm not sure that's something we can reverse. I think we might have to push through the commit with the error. @KTom101 @yangc95 @SSwiniarski Am I missing an alternative option to resolving this CLA issue?

**Comment by SSwiniarski** [comment]
> > @caupolicandiaz It seems that I commited with different username Ebrahim Al Aghbari for some reason and the CLA refuses to carry on unless its signed but I dont know how to remove that commit. Is it possible to remove that commit and them commit again with same user?
> 
> I'm not sure that's something we can reverse. I think we might have to push through the commit with the error. @KTom101 @yangc95 @SSwiniarski Am I missing an alternative option to resolving this CLA issue?

@caupolicandiaz We can merge with the error. We've done so with similar issues.

**Comment by E-Aghbari** [comment]
@SSwiniarski Thank you for merging my branch into the main. However, my name is does not appear and shows as Anonymous Contributor, though I linked my github account to my codecademy account. Is there a way to fix that to show up my name?

**Pull request merged**